### PR TITLE
build(nightly-build): update the key of the environment variable in the rule engine and upgrade the version number of kuiper

### PR DESCRIPTION
**Pull request by zhanghongtong**
update the key of the environment variable in the rule engine and upgrade the version number of kuiper

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-examples/blob/master/.github/CONTRIBUTING.md.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?

## Other information

### CODE REVIEW

**Comment by lenny-intel** [comment]
@zhanghongtong, please make these changes in the files under the `source` folder and then run `make build` from `source` folder to update these `generated` compose files. The version is now in the `source/.env` . The env vars for Kuiper are in `docker-compose-nexus-base.yml` & `docker-compose-nexus-add-mqtt.yml`

Also `chore` is not a valid type for your commit message & PR title. The type needs to adhere to the types defined in https://github.com/edgexfoundry/developer-scripts/blob/master/.github/Contributing.md#type
I suggest `build` for the type.

**Comment by zhanghongtong** [comment]
> @zhanghongtong, please make these changes in the files under the `source` folder and then run `make build` from `source` folder to regenerate those `generated` compose files you have manually edited.
> 
> All versions are now in the `source/.env` . The env vars changes for Kuiper need to be made in `docker-compose-nexus-base.yml` & `docker-compose-nexus-add-mqtt.yml`
> 
> Also `chore` is not a valid type for your commit message & PR title. The type needs to adhere to the types defined in https://github.com/edgexfoundry/developer-scripts/blob/master/.github/Contributing.md#type
> I suggest `build` for the type.

Hi, @lenny-intel 
Thank you for your reply, I have modified as per your request, please review

File: releases/nightly-build/compose-files/source/.env [hunk 1]
```diff
@@ -39,3 +39,4 @@ DEVICE_MODBUS_VERSION=master
 DEVICE_SNMP_VERSION=master
 DEVICE_MQTT_VERSION=master
 MOSQUITTO_VERSION=1.6.3
+KUIPER_VERSION=0.9.1-alpine
```
Comment (lenny-intel) [inline releases/nightly-build/compose-files/source/.env:42]: KUIPER_VERSION already exists on line 33. Just change it to new version

File: releases/nightly-build/compose-files/source/docker-compose-nexus-base.yml [hunk 2]
```diff
@@ -216,13 +216,12 @@ services:
...
-      KUIPER_REST_PORT: 48075
-      EDGEX_SERVICE_SERVER: http://edgex-core-data:48080
+      KUIPER__BASIC__CONSOLELOG: "true"
+      KUIPER__BASIC__RESTPORT: 48075
       EDGEX__DEFAULT__SERVER: edgex-app-service-configurable-rules
+      EDGEX__DEFAULT__SERVICESERVER: http://edgex-core-data:48080
       EDGEX__DEFAULT__TOPIC: events
       EDGEX__DEFAULT__PROTOCOL: tcp
       EDGEX__DEFAULT__PORT: 5566
-      EDGEX__DEFAULT__TYPE: zero
```
Comment (lenny-intel) [inline releases/nightly-build/compose-files/source/docker-compose-nexus-base.yml:226]: Why removing this?

File: releases/nightly-build/compose-files/docker-compose-nexus-no-secty.yml [hunk 3]
```diff
@@ -27,8 +27,10 @@ services:
   app-service-rules:
     container_name: edgex-app-service-configurable-rules
     depends_on:
-    - consul
-    - data
+      consul:
```
Comment (lenny-intel) [inline releases/nightly-build/compose-files/docker-compose-nexus-no-secty.yml:30]: These are new. I assume you didn't manually change these, correct? 
What version of docker compose do you have installed?

**Review by lenny-intel — CHANGES_REQUESTED** [review]
You also need to update `docker-compose-nexus-add-mqtt.yml` for the change in the environment variable names.

**Comment by tonyespy** [comment]
Just curious why all the condition_started statements were added as they're not mentioned in the commit message?

Also commit message summary lines are not suppose to wrap to into the commit message body, they're suppose to be a single (short) line which describes the commit. The body of the commit message is used to provide additional context, bugs, fixed, ...

File: releases/nightly-build/compose-files/docker-compose-nexus-no-secty.yml (hunk 3 above)
Comment (lenny-intel) [reply releases/nightly-build/compose-files/docker-compose-nexus-no-secty.yml:30]: @zhanghongtong , I have confirmed that there is a bug in the latest version of docker-compose (1.27.3) that is causing this incorrect generation of the `depends_on` section. Please downgrade to docker-compose version 1.27.2 and regenerate these compose files.

You can download the docker-compose 1.27.2 artifacts from here:
https://github.com/docker/compose/releases/tag/1.27.2

**Comment by lenny-intel** [comment]
> Just curious why all the condition_started statements

@tonyespy , see what I just posted in the Core slack channel.

File: releases/nightly-build/compose-files/source/docker-compose-nexus-base.yml (hunk 2 above)
Comment (zhanghongtong) [reply releases/nightly-build/compose-files/source/docker-compose-nexus-base.yml:226]: The default setting is `zero`, so I deleted this setting

**Comment by zhanghongtong** [comment]
> @zhanghongtong , please adjust your commit message and PR title per @tonyespy 's comment.
> 
> See guidelines here:
> https://github.com/edgexfoundry/developer-scripts/blob/master/.github/Contributing.md
Hi, @lenny-intel 
Please review again

**Review by lenny-intel — APPROVED** [review]
LGTM

**Pull request merged**